pulsar/consumer_partition.go

```diff
@@ -1680,6 +1696,9 @@ func (pc *partitionConsumer) internalClose(req *closeRequest) {
 }
 
 func (pc *partitionConsumer) reconnectToBroker(connectionClosed *connectionClosed) {
+	if pc.isSeeking.CompareAndSwap(true, false) {
```

Comment:
Wouldn't it be better to move to line 1747?
https://github.com/nodece/pulsar-client-go/blob/1f043f814406bc604314353467485eeeb9217e5e/pulsar/consumer_partition.go#L1745-L1750

Reply:
Once the connection is established, the broker will push the message to the client, so we need to reset the seek flag before the request reconnects.

Reply:
thank you for your explanation